test/rule-matches/no-autoplay-audio-matches.js

```diff
@@ -0,0 +1,41 @@
+describe('no-autoplay-audio-matches', function() {
```

Comment:
Add test case for media with duration <3 seconds and no loop attribute (should return false)

Reply:
This is tested in the check, rather than rule matches.
The rule matches fn is not aware of any allowedDuration & this is a part of the check.

Reply:
https://github.com/dequelabs/axe-core/pull/1946/files#diff-fbe82cba4f16ea544ec75aecbaedef3fR18-R23

Reply:
I still have not pushed my changes yet or dismissed your reviews. I am only recording my thoughts here. So hold off from commenting or reviewing until all my changes drop in.